src/types/upload/index.ts

```diff
@@ -68,14 +68,25 @@ export interface ProgressUpdater {
     setHasLivePhotos: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
+export interface ElectronFile {
```

Comment:
nit: For future, It would be nice if we can add some document the need for creating this new `ElectronFile` type.